src/index.ts

```diff
@@ -19,19 +17,20 @@ export default class FlatfileImporter extends EventEmitter {
 
   private apiKey: string
   private options: object
+  private customer: object
   private uuid: string
 
   private handshake: Penpal.IChildConnectionObject
 
   private $resolver: (data: any) => any
   private $rejecter: (err: any) => any
 
-  constructor (apiKey: string, options: object) {
+  constructor (apiKey: string, options: object, customer: object = {}) {
     super()
     this.apiKey = apiKey
     this.options = options
+    this.customer = customer
```

Comment:
Let's not instantiate with customer and let's add a `setCustomer` function

Reply:
There's an updateCustomer function, should we change it to setCustomer, or add a new setCustomer function?